packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -95,14 +92,10 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
   const layout = useFeature(feature.feedLayout);
   const { sidebarRendered } = useSidebarRendered();
   const { isFallback } = router;
-  useWindowEvents(
-    'popstate',
-    CHECK_POPSTATE,
-    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    useCallback(() => router.reload(), []),
-    { validateKey: false },
-  );
+
+  useEventListener(globalThis, 'popstate', () => {
```

Comment:
what did we use the validateKey and callback for again?

Reply:
Here it was not used because `validateKey: false`, on other places I moved checks to the callbacks. From what I seen the kratos `message` events had `data.eventKey` to differentiate so it was a guard to execute logic only for specific event.

Reply:
I see, ok no worries in that case.